cmd/node/list_test.go

```diff
@@ -0,0 +1,114 @@
+package node
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/cilium/cilium/api/v1/relay"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cilium/cilium/api/v1/observer"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_jsonOutput(t *testing.T) {
+	nodes := []*observer.Node{
```

Comment:
There are a certain number of fields in a node besides `Name` and `Version`. I think this test is a good fit for table-testing.